### PR TITLE
Movie list

### PR DESCRIPTION
**Pull request by AllmostHumann**
Hey! 

I have prepared movie list for you webpage ;). I have updated my branch with changes what you made before, and I did some changes with better folders layout,  some cosmetic changes in styles and I also removed from GlobalStyles the " display: flex;" option because the Header component didn't display at whole screen. 

image

Cheers!

### CODE REVIEW

File: src/features/movies/PopularMovies/index.js
Comment (alajurczak) [inline src/features/movies/PopularMovies/index.js:1]: Don't you think it should be index.js for the MovieList component? After that it doesn't have to be a component like PopularMovies.
If we are going to change it like this, MovieList should be in "feature".

**Review by alajurczak — APPROVED** [review]

**Review by PawelNackowski — CHANGES_REQUESTED** [review]
pages

In the 100% view, the tiles diverge to 3. Maybe use scaling so that they are always 4 and with a lower resolution, less?

**Comment by AllmostHumann** [comment]
> pages
> 
> In the 100% view, the tiles diverge to 3. Maybe use scaling so that they are always 4 and with a lower resolution, less?

**Comment by AllmostHumann** [comment]
> pages
> 
> In the 100% view, the tiles diverge to 3. Maybe use scaling so that they are always 4 and with a lower resolution, less?

Hey @PawelNackowski. I know there coudl be an issue with display the movie tiles. I made this kind of list made of a lot of <li> components just to show how it looks. When we implement API for movies it shoudl looks much better. Sorry I didn't let you know earlier about that. But if you want I can try to change this display right now.

**Comment by PawelNackowski** [comment]
It's ok. We'll fix it later.

**Comment by PawelNackowski** [comment]
> > pages
> > 
> > In the 100% view, the tiles diverge to 3. Maybe use scaling so that they are always 4 and with a lower resolution, less?
> 
> Hey @PawelNackowski. I know there coudl be an issue with display the movie tiles. I made this kind of list made of a lot of <li> components just to show how it looks. When we implement API for movies it shoudl looks much better. Sorry I didn't let you know earlier about that. But if you want I can try to change this display right now. 

It's ok. We'll fix it later.

**Comment by AllmostHumann** [comment]
> > > pages
> > > In the 100% view, the tiles diverge to 3. Maybe use scaling so that they are always 4 and with a lower resolution, less?
> > 
> > 
> > Hey @PawelNackowski. I know there coudl be an issue with display the movie tiles. I made this kind of list made of a lot of * components just to show how it looks. When we implement API for movies it shoudl looks much better. Sorry I didn't let you know earlier about that. But if you want I can try to change this display right now.
> 
> It's ok. We'll fix it later.

OK, perfect ;) I also implemetn @alajurczak  Pagination in my branch. So, if you agreed please approve my pull and I will merge it to main branch asap ;).

**Review by PawelNackowski — APPROVED** [review]

**Pull request merged**